bin/accountsandtransactions

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+require "atrium"
+
+::Atrium.configure do |config|
+  config.mx_client_id = "YOUR_MX_CLIENT_ID"
+  config.mx_api_key = "YOUR_MX_API_KEY"
+end
+
+puts "\n* Creating test user and member *"
+user = ::Atrium::User.create identifier: nil, is_disabled: nil, metadata: nil
+puts "Created user: " + user.guid
+
+credentialOne = {}
```

Comment:
This is a bit pedantic, but we should also be using snake case: `credential_one = {}`. Looks like we don't have rubocop enforcing the bin directory. That's ok. Once we move these, we can get rubocop added and it will automagically reformat these files.

Reply:
Oh, it looks like rubocop is already working. You can run  rubocop to auto correct: `bundle exec rubocop -a` to auto-format these files.